7-gce/bookshelf/templates/base.html

```diff
@@ -29,20 +29,7 @@
         </div>
         <ul class="nav navbar-nav">
           <li><a href="/books">Books</a></li>
-          <li><a href="/books/mine">My Books</a></li>
         </ul>
-        <p class="navbar-text navbar-right">
```

Comment:
Why was this code removed?

Reply:
The problem is that the My Books and Login features require OAuth. And our libraries require OAuth to have an endpoint registered through GCP Console auth. And the GCP Console auth screens require a top level domain (not a VIP). Two options, I think, leave it and force users to acquire a domain name to run the sample, or make the sample simpler. I opted for the latter.

Reply:
So does this change agree with, break, or is not related to https://cloud.google.com/appengine/docs/standard/python/authenticating-users-firebase-appengine?

Reply:
They're pretty independent, not related. Firebase auth is simpler, but designed mostly for AppEngine (just like before, it requires the (appspot) domain name). For these parts of Bookshelf that are non-AppEngine, like GCE, this auth will not work either. It'd be good for someone to perhaps work through Bookshelf on AE and port it to use Firebase auth so we don't have 2 samples showing 2 different ways to do the same thing. But, that's a different conversation and perhaps not worth it since people will want to do perhaps do one or the other equally.

Reply:
Ok.  I don't understand what this code was originally trying to do.  I'll ask Thea to confirm.

Reply:
Is this the relevant doc?
https://cloud.google.com/python/getting-started/authenticate-users

Reply:
Yes. The "Authorized redirect URIs" part is the issue on GCE login.

Reply:
Will that doc need to be updated if we remove this code?

Reply:
No. I believe you're copied on the CL for the docs, I did not change anything related to this. Login was not ever referenced in the GCE part of the tutorial. Probably another area the tutorials could be improved is being more directive how to explore/use the application. Anyway, no, this does not affect the tutorial, except to help users to successfully complete the GCE part of the tutorial.